scripts/modules/cli.py

```diff
@@ -10,6 +10,11 @@
 import sys
 import re
 
+try:
+    import yaml
+except ImportError:
+    print("No able to load yaml lib, use '--output-format json'")
```

Comment:
This seems a bit too noisy for folks without PyYAML, if they're only interested in JSON output to begin with.

I suggest moving `import yaml` under `if args.get("output_format") == 'yaml':`, so there's only error output if YAML output format is specified. Error message should be something like "Failed to import 'yaml': pip install yaml, or specify an alternative --output-format.".